From: Andrea Bolognani <eof@kiyuko.org>
Date: Sun, 22 Mar 2020 15:17:48 +0100
Subject: Fix man warning

This is reported by man(1) as

  mdoc warning: Empty input line #202

Spotted by Lintian.

Bug: https://github.com/conformal/spectrwm/pull/292
---
 spectrwm.1 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spectrwm.1 b/spectrwm.1
index 40eab4c..2d05f81 100644
--- a/spectrwm.1
+++ b/spectrwm.1
@@ -199,7 +199,7 @@ Any font defined here can then be selected in the status bar through the use of
 the markup sequence
 .Ic +@fn=n;
 where n is between 0 and 9.
-
+.Pp
 Also note that
 .Xr dmenu 1
 does not support Xft fonts.
